tests/packages/bugtool/test_bugtool.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+import subprocess
+
+# This smoke test runs xen-bugtool and verifies that the archive it generates
+# contains some of the expected logs and files.
+#
+# Requirements:
+# - an XCP-ng host
+
+class TestsBugtool:
+    def verify_contains(host, archive, files):
+        listing = host.ssh(['tar', '-jtf', archive])
+        listing = [x.split('/', maxsplit=1)[1] for x in listing.splitlines()]
+        all(file in listing for file in files)
```

Comment:
This doesn't raise, `all` only returns False if not all the conditions are met. I think the tests won't fail if a file is missing.